hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -221,6 +222,17 @@ public boolean isLockAcquisitionFailed() {
    */
   public final class DeleteCmdWorker implements Runnable {
```

Comment:
Not related to this PR, may be a future improvement, we can use [Scheduler](https://github.infra.cloudera.com/CDH/ozone/blob/3964ece6561b0402165c4158d51b65e9d82c8860/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/Scheduler.java#L32), rather than daemon and sleep.